setup.py

```diff
@@ -99,6 +99,7 @@
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: 3.13",
+        "Programming Language :: Python :: 3.14",
```

Comment:
```suggestion
        "Programming Language :: Python :: 3.14",
        "Programming Language :: Python :: Free Threading",
```
* https://pypi.org/classifiers -- or --

Programming Language :: Python :: Free Threading :: 1 - Unstable
Programming Language :: Python :: Free Threading :: 2 - Beta
Programming Language :: Python :: Free Threading :: 3 - Stable
Programming Language :: Python :: Free Threading :: 4 - Resilient